serviceLibrary/src/main/java/info/mqtt/android/service/MqttAndroidClient.kt

```diff
@@ -1289,7 +1291,7 @@ class MqttAndroidClient @JvmOverloads constructor(
 
     companion object {
         private val SERVICE_NAME = MqttService::class.java.name
-        private const val FOREGROUND_ID = 77
+        private const val FOREGROUND_ID = 77 shl 16
```

Comment:
Please can you explain why this is necessary and what's the purpose ?

Reply:
Sometimes the group notification has the same id. I think each time it is created, it creates a separate id.  Very little chance of polymerization, it can revert

Reply:
Yes, please revert. It's too much magic